core/client/client.go

```diff
@@ -121,7 +115,7 @@ func (c *Client) HasServerGroup(groupName string) bool {
 	return false
 }
 
-// MustEnsureClient will try to read the kube.yaml file from the host and connect to the cluster, if the Client or the Core Client is null.
+/*// MustEnsureClient will try to read the kube.yaml file from the host and connect to the cluster, if the Client or the Core Client is null.
```

Comment:
Why are we commenting this function? Can't we get rid of it if it's not being used?

Reply:
sorry, forget to delete it from code base